src/z64.h

```diff
@@ -60,13 +63,48 @@
 extern SDL_Surface *sdl_Screen;
 extern int screen_width, screen_height;
 
-typedef uint64_t UINT64;
-typedef int64_t INT64;
-typedef uint32_t UINT32;
-typedef int32_t INT32;
-typedef uint16_t UINT16;
-typedef int16_t INT16;
-typedef uint8_t UINT8;
-typedef int8_t INT8;
 typedef unsigned int offs_t;
 
+/*
+ * fast fix to missing <stdint.h> report
+ * These defs are not really as universal as the cxd4/rcp/my_types.h ones.
+ */
+#if (SCHAR_MIN <= -128 && SCHAR_MAX >= +127)
+typedef signed char INT8;
+typedef unsigned char UINT8;
+#else
+typedef signed int INT8;
+typedef unsigned int UINT8;
+#endif
+
+#if (SHORT_MIN <= -32768 && SHORT_MAX >= +32767)
```

Comment:
There's a problem unfortunately
```SHORT_MIN``` and ```SHORT_MAX``` are not defined on windows so I'm getting some compiler errors. I have to replace it with ```SHRT_MIN``` and ```SHRT_MAX``` but does this work on Linux?